### ISSUE TITLE
Enhancements to data science survey

### ISSUE THREAD

**Issue by claudiaregio** [microsoft/vscode-jupyter#6796]
We should add a button option that says "File Issue" on the data science survey that takes you to here: https://github.com/microsoft/vscode-jupyter/issues/new/choose

Had a chat with @rebornix about this the other day. Thought process here is we have a lot of people responding to the survey who have issues but most of the time they do not leave emails so we can follow up to get the issue solved. If a user is experiencing a blocking/frustrating issue, they might be more likely to click "File Issue" where we have a better chance of getting repro information instead of through the survey.

**Comment by rchiodo** [comment]
The idea here is that the 'File an Issue' is on the prompt, not at the survey. This hopefully gets more information on things people are commenting on in surveys.

**Comment by claudiaregio** [comment]
@rchiodo that is correct, idea was to have it be one of the "Yes", "No", "File Issue" options.

**Comment by rchiodo** [comment]
@claudiaregio did we add the blurb at the top? Was wondering if I should reopen this pending that.
